### ISSUE TITLE
Anchor links on high level tabs within components not working

### ISSUE THREAD

**Issue by MI6-94** [mi6/ic-design-system#237]
## Summary of the bug

You are unable to use a copied link from an anchor nav within the 'Code' or 'Accessibility' tabs to then take you back to that place. It loads the 'Guidance' tab.

## 🪜 How to reproduce

Tell us the steps to reproduce the problem:

1. Go to page: Footer component, 'Accessibility' tab.
2. Go to the heading 'For Assistive Technology'
3. Use the anchor to copy a link to that section (https://design.sis.gov.uk/components/footer#for-assistive-technology)
4. Paste it back into browser, it loads the 'Guidance' tab.

## 🧐 Expected behaviour

The link should take you to https://design.sis.gov.uk/components/footer#for-assistive-technology
